notebook/notebook/notebook.py

```diff
@@ -94,14 +123,15 @@ def start_pod(jupyter_token, image):
                 command=[
                     'jupyter',
                     'notebook',
+                    "--ip", "0.0.0.0", "--no-browser",
```

Comment:
I'm uncomfortable that the command is now stored in two places (the worker docker image and here). We should use `ENTRYPOINT`:
```
ENTRYPOINT ["jupyter", "notebook", "--ip", "0.0.0.0", "--no-browser"]
```
and then here we should replace command with:
```
args=[f'--NotebookApp.token...', ...]
```

Reply:
I see, I don't know enough about ENTRYPOINT, will read up.

Do you feel more comfortable with having one CMD shadow another, if they're decoupled (namely, cotton's local use of his notebook may not be quite the same as ours in the notebook service)?